PowerShellTools.HostService/ServiceManagement/Debugging/PowershellBreakpointRecord.cs

```diff
@@ -44,5 +44,35 @@ internal int Id
                 return _id;
             }
         }
+
+        public override bool Equals(object obj)
+        {
+            if (obj == null)
+            {
+                return false;
+            }
+
+            PowerShellBreakpointRecord record = obj as PowerShellBreakpointRecord;
+            if (record == null)
+            {
+                return false;
+            }
+            else
+            {
+                return record._id == this._id;
+            }
+        }
+
+        public override int GetHashCode()
+        {
+            if (_psBreakpoint != null)

```

Comment:
Not sure why I am checking if _psBreakpoint is null. The value of _id has nothing to do with the existence of _psBreakpoint.